plugins/declarative/src/voicecallhandler.cpp

```diff
@@ -20,7 +20,7 @@ class VoiceCallHandlerPrivate
 public:
     VoiceCallHandlerPrivate(VoiceCallHandler *q, const QString &pHandlerId)
         : q_ptr(q), handlerId(pHandlerId), interface(NULL), connected(false)
-        , duration(0), status(0), emergency(false), multiparty(false), forwarded(false)
+        , duration(0), status(0), emergency(false), multiparty(false), forwarded(false), initializing(false)
```

Comment:
Maybe with the uninitialized value (which would be default-initialized, anyway, so not really uninitialized) he meant to add explicit initialization here for `notifyInitializeError`?